src/modules/PublicLab.CustomInsert.js

```diff
@@ -112,5 +115,15 @@ module.exports = function CustomInsert(_module, wysiwyg) {
         });
       }
     });
+    // for the popover to disappear when clicked anywhere on the screen
+    $(':not(".woofmark-command-insert")').click((e) => {
+      if (popoverIsOpen && $('.woofmark-command-insert').children()[0] != e.target) {
+        const popoverContainer = document.querySelector('.popover');
+        const isChildElement = popoverContainer.contains(e.target);
+        if (popoverIsOpen && !e.target.classList.contains(".woofmark-command-insert") && !isChildElement) {
```

Comment:
```suggestion
        if (popoverIsOpen && !e.target.classList.contains("woofmark-command-insert") && !isChildElement) {
```

@RaviAnand111 That should fix the button not clicking issue

Reply:
ThankYou so much, I spent my whole day yesterday solving this bug, 
You solved it in a second, 
I am so grateful, how did you do it, I just want to be like you.
Thank You  @TildaDares @NARUDESIGNS

Reply:
😭

Reply:
I rectified the error @TildaDares 
Please have a look
Thank You